public/js/validations.js

```diff
@@ -14,4 +14,41 @@ function isTime(time) {
     } else {
         return false
     }
+}
+
+function isPhoneNumber(phone) {
+    let regEx = /^(([+][(]?[0-9]{1,3}[)]?)|([(]?[0-9]{4}[)]?))\s*[)]?[-\s\.]?[(]?[0-9]{1,3}[)]?([-\s\.]?[0-9]{3})([-\s\.]?[0-9]{3,4})$/gm
+    if (regEx.test(phone)) {
+        return true
+    } else {
+        return false
+    }
+}
+
+function isEmail(email) {
+    // email regex from http://emailregex.com - "Email Address Regular Expression That 99.99% Works."
+    let regEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+    if (regEx.test(email)) {
+        return true
+    } else {
+        return false
+    }
+}
+
+function isUrl(url) {
+    let regEx = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/gm
```

Comment:
should <http(s)> be optional or mandatory? On company's name card or something, many places omit http(s) in front of the website url.

Reply:
It's optional but we might consider trimming or enforcing it in the backend

Reply:
for now, without http(s), it doesn't pass the regex validation.
for instance something like (www.mayden.com) doesn't pass this.